src/main/java/org/apache/maven/doxia/cli/CLIManager.java

```diff
@@ -159,8 +164,12 @@ private static String getSupportedFormatAndEncoding()
 
     private static String getSupportedFormat()
     {
-        return "\nSupported Formats:\n from: " + join( DefaultConverter.SUPPORTED_FROM_FORMAT, ", " )
-            + " or autodetect" + "\n to:   " + join( DefaultConverter.SUPPORTED_TO_FORMAT, ", " )
+        String fromFormats = EnumSet.allOf( DefaultConverter.ParserFormat.class ).stream()
+                .map( f -> f.toString().toLowerCase() ).collect( Collectors.joining( ", " ) );
+        String toFormats = EnumSet.allOf( DefaultConverter.SinkFormat.class ).stream()
+                .map( f -> f.toString().toLowerCase() ).collect( Collectors.joining( ", " ) );
```

Comment:
Locale sensitive lower-case

Reply:
Doesn't make any difference (in any locale for ASCII characters used in the enum it is the same)